lib/mv_opentelemetry/live_view.ex

```diff
@@ -134,4 +141,9 @@ defmodule MvOpentelemetry.LiveView do
     OpentelemetryTelemetry.end_telemetry_span(opts[:tracer_id], meta)
     :ok
   end
+
+  defp namespace_key({key, value}, prefix) when is_binary(key) do
```

Comment:
Could you move this slightly higher - underneath the last function where you call it (line 86 or something) - now there's a big gap between an implementation and a call stack.

Reply:
Another suggestion is to rename it to something like `prefix_with`, when reading the callee i see namespace and it doesn't tell me much (i.e. namespace how exactly)

Reply:
I cannot move it any higher than it is without creating a compilation warning, it's right below many handle_events :)